scripts/create-hub-access-kubeconfig.sh

```
@@ -21,7 +21,7 @@ token=$(oc -n "$namespace" extract secret/hub-access --keys token --to - 2>/dev/
 
 echo "generating a kubeconfig for hub-access serviceaccount in $namespace namespace on $server_url"
 
-cat <<EOF >/tmp/kubeconfig.hub-access
+cat <<EOF >/tmp/kubeconfig.hub-access.${server_name}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)